src/utils/helpers.ts

```diff
@@ -1,7 +1,8 @@
 import fs from "fs";
+const brake = require("brake");
```

Comment:
Imports in typescript should be:
```suggestion
import brake from "brake";
```
If you get type issues try run `yarn add -D @types/brake`.